README.md

```diff
@@ -35,6 +35,7 @@ make backwards-incompatible changes.
 | [Bigtable][cloud-bigtable]                      | stable | [`cloud.google.com/go/bigtable`](https://pkg.go.dev/cloud.google.com/go/bigtable)                                             |
 | [Cloudbuild][cloud-build]                       | stable | [`cloud.google.com/go/cloudbuild/apiv1`](https://pkg.go.dev/cloud.google.com/go/cloudbuild/apiv1)                             |
 | [Cloudtasks][cloud-tasks]                       | stable | [`cloud.google.com/go/cloudtasks/apiv2`](https://pkg.go.dev/cloud.google.com/go/cloudtasks/apiv2)                             |
+| [Compute Engine][cloud-compute]                 | beta   | [`cloud.google.com/go/compute/apiv1`](https://pkg.go.dev/cloud.google.com/go/compute/apiv1)                                                                                      |
```

Comment:
I generated it as `alpha`. Should it be `beta`?

https://github.com/googleapis/google-cloud-go/blob/2a43d6d30d7041eb6ed0b305c81dc32c8c42ebc1/internal/gapicgen/generator/config.go#L67-L68